sovrin_client/agent/walleted.py

```diff
@@ -97,6 +102,17 @@ def __init__(self,
         }
         self.logger = logger
 
+        self.issuer_backend = None
+
+        self._invites = {}  # type: Dict[Nonce, Tuple(InternalId, str)]
+        self._attribDefs = {}  # type: Dict[str, AttribDef]
+        self.defined_claims = []  # type: List[Dict[str, Any]
+
+        self.available_claims_by_internal_id = {}  # type: Dict[InternalId, Set[ID]]
+
+        # dict for proof request schema Dict[str, Dict[str, any]]
+        self._proofRequestsSchema = {}
```

Comment:
As I mentioned in one of the earlier comment as well, this state is also there in in sovrin-client/sovrin_client/anon_creds/sovrin_issuer.py file file.